app/forms/admin/move_form.rb

```diff
@@ -0,0 +1,91 @@
+# frozen_string_literal: true
+
+module Admin
+  # Admin form object for moving a work to a different collection.
+  class MoveForm < ApplicationForm
+    before_validation :normalize_druid
+    attribute :collection_druid, :string
+    validate :validate_collection_is_targetable
+
+    attribute :content_id, :integer
+    attribute :work_form
+
+    def collection
+      @collection ||= Collection.find_by(druid: collection_druid)
+    end
+
+    def normalize_druid
+      return if collection_druid.starts_with?('druid:') || collection_druid.blank?
+
+      self.collection_druid = collection_druid.prepend('druid:')
+    end
+
+    def validate_collection_is_targetable
+      return if validate_collection_blank
+
+      return if validate_collection_found
+
+      return if validate_already_part_of_collection
```

Comment:
It looks like these methods conditionally add an error but all return a boolean. I wondered about renaming these to be question-mark methods since we use them to determine validation flow. Would it be surprising for a question-mark method to also add an error, though? (Maybe not since we're in a form validation context?)

E.g.:

```suggestion
      return if collection_druid_missing? || collection_not_found? already_part_of_collection?
```

Reply:
@justinlittman I buy that. I might still like the `validate` method names to be tweaked. Right now, it's not clear whether `validate_collection_blank` and `validate_collection_found` are consider "blank" and "found" to be the valid conditions or the invalid ones. Especially because the former marks "blank" invalid and "found" valid. The question mark method names above make the intent clearer and read more like English to me, at least. We could rename the methods to those names but remove the question marks?

Reply:
@justinlittman I like this even better, thanks!